tools/include-env-compressed/tests/include_build_script.rs

```diff
@@ -0,0 +1,29 @@
+use include_env_compressed::ArchiveKind;
+
+const BUILD_SCRIPT_CONTENTS: &str = r#"
```

Comment:
why not use `include_bytes!` here or otherwise read the file contents to a string as a `lazy_static` or in the test? seems brittle to have a copy of the file contents as a literal string in the test.